model/canvas.cc

```diff
@@ -395,8 +395,20 @@ namespace minsky
   
   ItemPtr Canvas::itemAt(float x, float y)
   {
-    auto item=model->findAny(&Group::items,
-                        [&](const ItemPtr& i){return i->visible() && i->contains(x,y);});
+    // Fix for library dependency problem with items during Travis build     
+    ItemPtr item;                    
+    auto minD=numeric_limits<float>::max();
+    model->recursiveDo(&GroupItems::items,
```

Comment:
Great - almost exactly how I would have done it! An it looks like it has fixed the problem - yay!

Reply:
Thanks. I just followed your instructions, which were crystal clear, and repurposed your existing code in canvas.cc for finding the closest "in" port. I wish all bugs were this easy to fix.:)